ppa/archive/models.py

```diff
@@ -707,9 +725,12 @@ def index_data(self):
         if self.status == self.SUPPRESSED:
             return {"id": self.source_id}
 
+        index_id = self.index_id()
         return {
-            "id": self.source_id,
+            "id": index_id,
             "source_id": self.source_id,
+            "first_page_i": list(self.page_span)[0] if self.pages_digital else None,
```

Comment:
can this just call `first_page()`? seems like the same logic

Reply:
good catch. (I'm gonna say the method didn't exist yet when I first wrote this .... but who knows)